web/test/tests/IndexIntegrationTest.java

```diff
@@ -67,6 +67,7 @@ public static Collection<Object[]> data() {
 			{ "publication.location:Berlin AND publication.startDate:1993", /*->*/ 1 },
 			{ "publication.location:Berlin AND publication.startDate:[1992 TO 2017]", /*->*/ 5 },
 			{ "inCollection.id:\"http\\://lobid.org/organisations/DE-655#\\!\"", /*->*/ 133 },
+            { "inCollection.id:\"https\\://nrw.digibib.net/search/hbzvk/\"", /*->*/ 150 },
```

Comment:
Wrong indentation. Should be three tabs (`\t`)?

Reply:
I thought we changed it to spaces. I am not sure why we still have all these tabs as indentation in the java files.
@dr0i could you check. I opened a ticket for that: https://github.com/hbz/lobid-resources/issues/2023

For now I changed the indentation to tabss.